src/main/java/me/hugeblank/allium/lua/type/UserdataFactory.java

```diff
@@ -188,8 +188,16 @@ public static Object toJava(LuaState state, LuaValue value, Class<?> clatz) thro
         if (clatz.isPrimitive()) {
             if (value.isInteger() && clatz.isAssignableFrom(int.class)) { // int
                 return value.toInteger();
+            } else if (value.isInteger() && clatz.isAssignableFrom(byte.class)) { // double
```

Comment:
either change or remove the comments here. I don't really think we need them anymore, but if they're going to be there, it's best to be accurate.